Contributing/README.md

```diff
@@ -0,0 +1,54 @@
+# Contributing
+
+This section contains important information and a list of resources we would like to be followed while contributing to moja global. It consists of ways that you can use to communicate with us, contribute to our development and processes and join our organization and strategy board. We are increasingly looking for developers, scientists, researchers to assist us with our new projects, provide mentorship wherever required and help us maintain our existing projects. 
+
+## Getting Started
+
+Help fight climate change from your keyboard! Moja global provides tools for estimating emissions and removals of greenhouse gases (GHG) from the land sector. Our flagship software, Full Lands INtegration Tool is also known as FLINT is based on over 20 years of experience building and operating measurement, reporting and verification (MRV) tools in Australia and Canada. 
+
+The FLINT provides the capability to quickly establish an operational MRV system that can respond to policymaking and planning needs. The FLINT has been designed to achieve the required accuracy through step-by-step improvements. The resources below help you quickly identify how you can get started with contributing to our processes, tips, tricks and suggestions from our side, including how you can reach out to us.
+
+### Community Contributions
+
+-   [How to join moja global?](How-to-Join-moja-global.md)
+-   [How to join Strategy Board?](How-to-Join-the-Strategy-Board.md)
+-   [How to provide strategic advice?](How-to-Provide-Strategic-Advice.md)
+-   [How to organize events?](How-to-Organise-Events.md)
+
+### Code Contributions
+
+-   [How to contribute code?](How-to-Contribute-Code.md)
+-   [How to start a new project?](How-to-Start-a-New-Project.md)
+-   [How to create a repository?](How-To-Create-a-Repository.md)
+-   [How to develop a test?](How-to-Develop-a-Test.md)
```

Comment:
How-to-Develop-a-Test.md does not exist.

Reply:
I guess it does. It works on the GitHub preview and locally as well. Can you please check?

Reply:
It does! Resolving this conversation!